book/website/community-handbook/consistency/consistency-formatting.md

```diff
@@ -0,0 +1,194 @@
+
+
+(ch-consistency-formatting-hr-labels)=
+### Check 3: Add labels to chapters, subchapters, sections, and images to enable cross-referencing.
+
+Often, a chapter might refer to content from another chapter to explain concepts or expand on points.
+Cross-referencing facilitates this by ensuring that the referred content is easy to find with a simple click.
+This helps make _The Turing Way_ more navigable and accessible.
+
+{ref}`Cross-referencing <ch-style-crossref>` is discussed in detail in the {ref}`ch-style-guide`. The subchapter explains what labels are, provides a naming convention for labels in _The Turing Way_, and gives several useful examples for how cross-referencing should be done.
+
+
+(ch-consistency-formatting-hr-images)=
+### Check 4: Use `MyST` for image formatting
+
+Some figures and images in _The Turing Way_ are embedded using Markdown syntax.
+While this works, it does not allow the images to adapt to the screen size of the device the book is read from. 
+
+Markedly Structured Text (`MyST`) is a flavour of Markdown that addresses this and enables responsive images in the _The Turing Way_.
+
+It also allows the use of captions and alternative text (ALT text), which are the invisible image descriptions that are read aloud to readers of the _The Turing Way_ who use a screen reader.
+If no ALT text is provided with an image, these users will be unable to understand the purpose of the image.
+
+When writing ALT text, remember to:
+- **Be descriptive** - Adequately describe the image using its content and context for guidance. 
+In doing so, there is no need to "announce" an image in your description (for example, using "illustration of" or "picture of") since screen readers will already do this.
+- **Keep it as short as possible** - Although a long description may be necessary for some images, it is better to keep them as short as possible. 
+This ensures that the descriptions are easy to understand.
+
+Please note that images included in _The Turing Way_ book should be less than 1MB.
+This allows the book load faster, especially for readers who may have slow internet connections.
+
+Please refer to the {ref}`style guide <ch-style-figures>` for examples on formatting images using `MyST` and adding ALT text to them.
+When including images in your contributions, it may be better to avoid the height parameter as the wrong value could make your image appear distorted on mobile devices.
+You should always check how your image looks in the Netlify preview of the book when you make a PR.
+
+(ch-consistency-formatting-sr)=
+## Soft Requirements
+
+Soft requirements help improve the overall look and feel of _The Turing Way_.
+When effected, these checks may go unnoticed, but they also make _The Turing Way_ a polished piece of work.
```

Comment:
This sentence could be clearer, it suggests the changes are unnoticable but make things polished. It's a bit contradictory again?